es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 **1. Machine Learning tips and tricks cheatsheet**
 
-&#10230;
+&#10230; Hoja de trucos y ayudas de aprendizaje automático
 
 <br>
 
 **2. Classification metrics**
 
-&#10230;
+&#10230; Métricas de clasificación
 
 <br>
 
 **3. In a context of a binary classification, here are the main metrics that are important to track in order to assess the performance of the model.**
 
-&#10230;
+&#10230; En el contexto de una clasificacion binaría, estas son las principales métricas que se tienen que seguir para evaluar el desempeño del modelo
 
 <br>
 
 **4. Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
 
-&#10230;
+&#10230; Matriz de confusión ― La matriz de confusión es usada para tener un mejor panorama cuandpo se evalua el desempeño del modelo. Se define como:
```

Comment:
You misspelt _cuando_ as _cuandpo_. Add the accent in _evalúa_. Again, I would translate "performance" as _rendimiento_.